tests/LoggerTest.php

```diff
@@ -181,5 +182,12 @@ public function testAdapters(): void
         $logger = new Logger($adapter);
         $response = $logger->addLog($log);
         $this->assertEquals(200, $response);
+
+        // Test Bugsnag
+        $bugsnagKey = \getenv('TEST_BUGSNAG_KEY');
+        $adapter = new Bugsnag($bugsnagKey ? $bugsnagKey : '');
+        $logger = new Logger($adapter);
+        $response = $logger->addLog($log);
+        $this->assertEquals(200, $response);
```

Comment:
I tried running this test and it passed, but I don't see any new errors in bugsnag. Were you able to see something in bugsnag?

Reply:
@stnguyen90 I didn't see anything in Bugsnag, as I don't recall sending any errors to Bugsnag. Do you know how we can send errors to Bugsnag to validate these changes? I'm willing to help in any way I can.

Reply:
@michizhou, you can sign up for a trial at [BugSnag](https://www.bugsnag.com/).